spec/models/ldap_request_helper_spec.rb

```diff
@@ -9,48 +9,54 @@ class TestLdapRequestHelper
   let(:test_ldap_conn) { LdapRequestHelper::LdapConnection.new }
   let(:ldap_entry) { FactoryGirl.build(:ldap_entry) }
 
+  # The following is commented out because we are switching to Github Actions
+  # and GA does not run in a whitelisted network.
   context "provides a valid ldap interface" do
```

Comment:
```suggestion
  xcontext "provides a valid ldap interface" do
```

This should mark the tests as "pending" which would allow CI to pass but help document that the specs or the CI environment need to be addressed.